cfripper/cli.py

```diff
@@ -138,6 +145,11 @@ def process_template(
     help="Logging level",
     show_default=True,
 )
+@click.option(
+    "--rules-config-file",
+    type=click.Path(exists=True, resolve_path=True, readable=True, file_okay=False),
+    help="Loads rules configuration file",
```

Comment:
Maybe make the help text slightly more explicit? - `Loads rules configuration file for filtering (type: [.py, .pyc])`.